onnxruntime/hosting/http/http_server.cc

```diff
@@ -29,42 +25,44 @@ namespace hosting {
 using handler_fn = std::function<void(std::string, std::string, std::string, HttpContext&)>;
 
 App::App() {
-  // TODO: defaults should come from central place
-  address_ = boost::asio::ip::make_address_v4("0.0.0.0");
-  port_ = 8080;
-  threads_ = std::thread::hardware_concurrency();
+  http_details.address = boost::asio::ip::make_address_v4("0.0.0.0");
+  http_details.port = 8080;
+  http_details.threads = std::thread::hardware_concurrency();
 }
 
 App& App::Bind(net::ip::address address, unsigned short port) {
-  address_ = std::move(address);
-  port_ = port;
+  http_details.address = std::move(address);
+  http_details.port = port;
   return *this;
 }
 
 App& App::NumThreads(int threads) {
-  threads_ = threads;
+  http_details.threads = threads;
```

Comment:
Only assign `http_details.threads` when `threads > 0`?

Reply:
We check that in configuration